deltachat-ios/Chat/ChatViewController.swift

```diff
@@ -1202,31 +1206,72 @@ class ChatViewController: UITableViewController, UITableViewDropDelegate {
         }
     }
 
+    @available(iOS 14, *)
+    private func clipperButtonMenu() -> UIMenu {
+        func action(localized: String, systemImage: String, attributes: UIMenuElement.Attributes = [], handler: @escaping () -> Void) -> UIAction {
+            UIAction(title: String.localized(localized), image: UIImage(systemName: systemImage), attributes: attributes, handler: { _ in handler() })
+        }
+        var actions = [UIMenuElement]()
+        actions.append(action(localized: "camera", systemImage: "camera", handler: showCameraViewController))
+        let galleryImage = if #available(iOS 16, *) { "photo.stack" } else { "photo" }
+        actions.append(action(localized: "gallery", systemImage: galleryImage, handler: showPhotoVideoLibrary))
+        actions.append(action(localized: "files", systemImage: "folder", handler: showDocumentLibrary))
+        if dcContext.hasWebxdc() {
+            actions.append(action(localized: "webxdc_apps", systemImage: "app", handler: showWebxdcSelector))
+        }
+        actions.append(action(localized: "voice_message", systemImage: "waveform", handler: showVoiceMessageRecorder))
```

Comment:
```suggestion
        actions.append(action(localized: "voice_message", systemImage: "mic", handler: showVoiceMessageRecorder))
```

this is the icon we're using on Android, i think, it makes sense to stay in sync, esp. when it is so easy :)

for the name: "SF Symbol" says, if one needs to support platforms older than iOS 18, one shall use "mic" instead of "microphone":

![Screenshot 2025-01-16 at 14 35 02](https://github.com/user-attachments/assets/687c2886-c81b-45c7-832b-82760e76a363)